p2p/simulations/cmd/p2psim/main.go

```diff
@@ -0,0 +1,425 @@
+// p2psim provides a command-line client for a simulation API.
```

Comment:
I thought it best to keep all the simulation stuff under `p2p/simulations` (people won't want to build this binary unless they are running a simulation), I can move if you feel strongly though.